src/app/components/left-menu/left-menu.component.ts

```diff
@@ -115,7 +119,8 @@ export class LeftMenuComponent {
   }
 
   public save(event) {
-    if (this.persistenceService.isAvailable) {
+    if (this.persistenceService.isAvailable
+      || (this.persistenceService.isAvailable && this.persistenceService.isAuthenticate && this.persistenceService.isAuthAvailable)) {
```

Comment:
if `this.persistenceService.isAvailable` is true then the second part after || is true and same thing if it is false

Reply:
Fixed !
Too many conditions kill the condition